### ISSUE TITLE
Path expanding warning when running as system service

### ISSUE THREAD

**Issue by jodal** [mopidy/mopidy#1249]
When running Mopidy 1.1.0 as a system service, you typically don't have the `$XDG_*` vars defined in the environment. Thus you get the following warning when expanding the `file/media_dirs` default config value.

```
WARNING  Failed expanding path ($XDG_MUSIC_DIR) fromfile/media_dirs config value.
```

I haven't decided on a solution here, but I know that I don't want this to scare people looking at the system service logs.

**Comment by adamcik** [comment]
I think `XDG_MUSIC_DIR` is one of the XDG dirs that might not always be defined. So maybe just use a fallback?

**Comment by adamcik** [comment]
Or we could have a `--system` flag to let us know what context we are running in?

**Comment by trygveaa** [comment]
FWIW, I have a pretty slim setup on my machine (no DE), and I don't have set any of the XDG variables we use. (The only ones I have are XDG_RUNTIME_DIR, XDG_SEAT, XDG_SESSION_ID and XDG_VTNR.)

**Comment by adamcik** [comment]
To clarify a bit I was thinking about http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html which has the base values, that all have well established fallbacks, while the xdg user dir ones like music don't as far as I can remember.

**Comment by jodal** [comment]
Looking closer at this specific case, I think it is fair to simply make the log message a debug level message instead. The default value for `file/media_dirs` is:

```
media_dirs =
    $XDG_MUSIC_DIR|Music
    ~/|Home
```

These are good defaults for most users. If a path here cannot be expanded because of missing `$XDG_*` variables, it's probably one of our defaults failing, a not a user that is using `$XDG_*` vars when setting the config themselves. Thus debug level makes sense.

Though, if we *can* expand the path and it doesn't exist, then I think we should assume it is a user that has typoed some path or similar, and keep logging at warning level. I'll give that log message a bit more context.

**Comment by jodal** [comment]
Fixed by #1255.
